README.md

```diff
@@ -198,6 +198,31 @@ Add following to `Cartfile`:
 github "RxSwiftCommunity/RxAlamofire" "master"
 ```
 
+### Swift Package manager
+
+Create a `Package.swift`  file
+
+```
+// swift-tools-version:4.0
+
+import PackageDescription
+
+let package = Package(
+name: "TestRxAlamofire",
+
+dependencies: [
+    .package(url: "https://github.com/RxSwiftCommunity/RxAlamofire.git", from: "4.0.0"),
+],
+
+targets: [
+.target(
```

Comment:
This looks mis-formatted, do you mind formatting this correctly with 4-spaces preferably ?